docs: update posts with multiple options in app (#730)
devopness/devopness: docs/environments/list-environments.md

---
title: List Environments

> intro: Access the list of project environment to see details of an environment.

Reviewer: ```suggestion
intro: You can find and view details of project environments you have access to
```

Author: intro updated to better describe the post objective
